doc/INSTALL.md

```diff
@@ -99,3 +99,32 @@ $ nix-shell -Q -p gdb sqlite autoconf git clang libtool gmp sqlite autoconf \
 autogen automake 'python3.withPackages (p: [p.bitcoinlib])' \
 valgrind asciidoc --run make
 ```
+
+To Cross-compile for Android
```

Comment:
Nit: "To cross-compile for Android" – lower-case "cross-compile" :-)

Reply:
Fixed :)